src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -139,7 +139,101 @@ def list_dsn_members(self, dataset_name, member_pattern=None,
         custom_args["headers"]["X-IBM-Attributes"] = attributes
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json['items']  # type: ignore
-
+    
+    def copy_uss_to_dataset(self,from_filname,to_dataset_name,to_member_name="",type="text",replace=False):
+        """
+        Copy a USS file to dataset 
+            Parameters
+            ----------
+            from_filname: str
+                Name of the file to copy from
+            to_dataset_name: str
+                Name of the dataset to copy to
+            to_member_name: str
+                Name of the member to copy to  
+            type: str
+                Type of the file to copy from
+            enq: str
+                Enqueue type for the file to copy from
+            replace: bool
+                If true, the function uses the REPLACE=YES on ARCHDEF request, otherwise it uses the REPLACE=NO.
+            Returns 
+            -------
+            json
+                A JSON containing the result of the operation
+        """
+        
+        data={
+            "request":"copy",
+           "from-file":{
+               "filename":from_filname.strip(),
+                "type":type.strip()
+           },
+           "replace":replace
+        }
+        
+        path_to_member = f"{to_dataset_name.strip()}({to_member_name.strip()})" if to_member_name else to_dataset_name.strip()    
+        custom_args = self._create_custom_request_arguments()
+        custom_args['json'] = data
+        custom_args["url"] = "{}ds/{}".format(self.request_endpoint, path_to_member)
+        response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[200])
+        return response_json
+    
+    def copy_dataset_or_member(self,from_dataset_name,to_dataset_name,from_member_name="",volser="",alias="",
+                               to_member_name="",enq="",replace=False):
```

Comment:
For the default value of `from_member_name`, `to_member_name`, `volser`, `alias`, and `enq`, can we use `None` instead of an empty string?